.github/workflows/version-or-publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Version or Publish
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  changesets:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+
+      - name: Setup Node.js 12.x
+        uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+
+      - name: Install Dependencies
+        run: yarn
+
+      - name: Create Release Pull Request or Publish
+        id: changesets
+        uses: changesets/action@master
+        with:
+          version: yarn version
+          publish: yarn release
```

Comment:
Can you explain why do we need those two custom built scripts? 🤔

Reply:
version:
- runs changeset version
- and updates README

release:
- checks if the current version has already been released
- builds
- commits
- tags the commit
- pushes the tag out

It's not a standard npm-based flow that is somewhat assumed by Changesets.